kubernetes_asyncio/client/rest.py

```diff
@@ -137,13 +137,15 @@ async def request(self, method, url, query_params=None, headers=None,
 
         # For `POST`, `PUT`, `PATCH`, `OPTIONS`, `DELETE`
         if method in ['POST', 'PUT', 'PATCH', 'OPTIONS', 'DELETE']:
-            if re.search('json', headers['Content-Type'], re.IGNORECASE):
+            if (re.search('json', headers['Content-Type'], re.IGNORECASE) or
```

Comment:
This file is generated by the generator. Could you update scripts/rest_client_patch.diff? This patch is applied after generating files.

Reply:
Done